### PR TITLE
Add SR-IOV Device Plugin CDI support

### PR DESCRIPTION
**Pull request by e0ne**

### CODE REVIEW

**Review by SchSeba — COMMENTED** [review]
Hi @e0ne should this be part of the sriovOperatorConfig and not env variable?

File: bindata/manifests/plugins/sriov-device-plugin.yaml [hunk 1]
```diff
@@ -69,6 +72,12 @@ spec:
           readOnly: true
         - name: device-info
           mountPath: /var/run/k8s.cni.cncf.io/devinfo/dp
+        {{- if .UseCDI }}
+        - name: dynamic-cdi
+          mountPath: /var/run/cdi
+        - name: host-config-volume
```
Comment (adrianchiris) [inline bindata/manifests/plugins/sriov-device-plugin.yaml:78]: why  do we need this one ?

Comment (adrianchiris) [reply bindata/manifests/plugins/sriov-device-plugin.yaml:78]: ack, so once removed im LGTM with this one.

**Review by adrianchiris — APPROVED** [review]
please update crd in helm chart as well. otherwise LGTM


Update: NVM its there already :)

**Review by zeeke — APPROVED** [review]
LGTM

Ready to merge if CI is green

**Pull request merged**